pkg/libinit/proc_linux.go

```diff
@@ -41,6 +48,31 @@ func WithTTYControl(ctty bool) CommandModifier {
 	}
 }
 
+func WithMultiTTY(mtty bool, openFn func([]string) ([]*os.File, error), ttyNames []string) CommandModifier {
+	return func(c *exec.Cmd) {
+		if mtty {
+			ww, err := openFn(ttyNames)
+			if err != nil {
+				return
+			}
+
+			if len(ww) >= 1 {
```

Comment:
I had to read this twice; why not just len(ww) > 0

Reply:
Changed to include a switch/case for situation with no or just one tty defined.